docs/Developers/Profile_Bears.rst

```diff
@@ -0,0 +1,74 @@
+Profile Bears
+=============
+
+This document provide the overview of coala's Profiling Interface. The
+Profiling Interface will provide an ability to Profile the Bear's code to
+optimize its performance.
+
+The profiler will start by profiling the run() method of bears because it
+consumes most of the bears time. So, this is part where bear writer will spend
```

Comment:
You should omit the part "consumes most the bears time", that's an assumption that is usually satisfied for larger setups.
Also you actually describe the real reason below :)
-->
The profiler will start by profiling the run() method of bears because ~~it
consumes most of the bears time. So,~~ this is *the* part where bear writer**s** will spend time on ...